examples/simulation/TensorBoardStrategy.py

```diff
@@ -0,0 +1,58 @@
+import flwr as fl
+import tensorflow as tf
+from typing import Optional
+from typing import List, Dict, Optional, Tuple
+from flwr.common import Scalar
+import os, os.path
+
+
+def weighted_loss_avg(results: List[Tuple[int, float, Optional[float]]]) -> float:
+    """Aggregate evaluation results obtained from multiple clients."""
+    num_total_evaluation_examples = sum(
+        [num_examples for num_examples, _, _ in results]
+    )
+    weighted_losses = [num_examples * loss for num_examples, loss, _ in results]
+    return sum(weighted_losses) / num_total_evaluation_examples
+
+
+class TensorBoardStrategy(fl.server.strategy.FedAvg):
+    def aggregate_evaluate(
+        self, rnd: int, results, failures,
+    ) -> Tuple[Optional[float], Dict[str, Scalar]]:
+        """Aggregate evaluation losses using weighted average."""
```

Comment:
After thinking about it a while I was wondering if this could be written as a wrapper for a Strategy instance hooking into the `aggregate_evaluate` or as a simple function which everyone can use in their `aggregate_evaluate` and just pass the `aggregate_evaluate` arguments to it.

Reply:
@tanertopal I was also thinking about writing a utils file for a function and put in common folder.